### ISSUE TITLE
Prepare for video visit page - Accessibility Issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16846]
## What Happened?
**VoiceOver / TalkBak**

1. With the appropriate screen reader enabled on iOS and Android, the a11yHint for the back button is not being picked up 
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)
2. With TalkBack enabled on Android, the programmatic title is incorrectly set

## Specs:
- Device: Android -Google Pixel 3XL(v.11) with TalkBack enabled
- Device: iOS - iPhone XR (v.14.2) with VoiceOver enabled
- User: marcy.nadeau@va.gov

## Steps to Reproduce
**VoiceOver / TalkBak**

For 2:
1. With Talkback enabled on Android, navigate to the prepare for a video visit page
2. Select the page
3. Note that the programmatic title is picked up as "Prepare for video visit screen" instead of "Prepare for video visit page"

## Desired Behavior
**VoiceOver / TalkBak**

For 2:
- With TalkBack enabled on Android, the programmatic title will be picked up as “"Prepare for video visit page"

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 4 - Low

## Linked Story (s)
- Story #15761

## Screen Shot (s) and Additional Information

**Comment by digitalken** [comment]
@komaly15 
Fail 1/14/2021 - Still announcing "Prepare for video visit screen" rather than "Prepare for video visit page" on Android.  Sending back to "in progress."

**Comment by digitalken** [comment]
PASS 1/25/2021 Closing.

**Comment by bessgreen** [comment]
Ticket to update a11y hints to match page headers: #18951
